src/linalg.jl

```diff
@@ -110,10 +111,16 @@ Normalized Laplacian matrix of graph `g`.
 - `T`: result element type of degree vector; default is the element type of `g` (optional).
 - `selfloop`: adding self loop while calculating the matrix (optional).
 """
+function normalized_laplacian(sg, T::DataType=eltype(sg); selfloop::Bool=false)
```

Comment:
So............is this what you mean the relax of type signature here?

Reply:
Yeah, because Zygote would freak out that there was no method for `RefValue`s of graphs or matrices